src/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>to do list</title>
+</head>
+
+<body>
+    <div id="container">
+        <div id="today">
+            <p id="title">Today's To Do</p>
```

Comment:
Based on HTML best practice, it's recommended to have at least on `h1` to `h6` tag in the document, so please consider replacing this `p` tag to a `h` tag.

Reply:
Thank you. I am fixing it right away! 👍